h/templates/accounts/login.html.jinja2

```diff
@@ -8,6 +8,8 @@
 
 {% block content %}
   <div class="form-container content">
+    {% include "h:templates/includes/flash-messages.html.jinja2" %}
```

Comment:
Add the flash messages to the login page, so that they actually get rendered there